marklogic-data-hub-central/ui/src/components/expandable-table-view/expandable-table-view.tsx

```diff
@@ -72,29 +73,37 @@ const ExpandableTableView: React.FC<Props> = (props) => {
 
   const columns = [
     {
-      title: "Property",
-      dataIndex: "property",
-      width: "20%",
+      text: "Property",
+      headerFormatter: () => <span aria-label="property-header">Property</span>,
+      dataField: "property",
+      width: 200,
     },
     {
-      title: "Value",
-      dataIndex: "value",
-      width: "20%",
+      text: "Value",
+      headerFormatter: () => <span aria-label="value-header">Value</span>,
+      dataField: "value",
+      width: 200,
     },
     {
-      title: "View",
-      dataIndex: "view",
-      width: "20%",
+      text: "View",
+      headerFormatter: () => <span aria-label="view-header">View</span>,
+      dataField: "view",
+      width: 200,
     }
   ];
 
   return (
-    <Table
+    <HCTable
+      className="expandable-table-view"
       rowKey="key"
-      dataSource={data}
       columns={columns}
+      data={data}
       pagination={false}
       data-cy="expandable-table-view"
+      subTableHeader={false}
+      showExpandIndicator={{bordered: true}}
+      childrenIndent={true}
+      nestedParams={{headerColumns: columns, state: [expandedNestedRows, setExpandedNestedRows]}}
```

Comment:
Can we add `showHeader={true}` here since my PR got merged and added this property? The header will not be displayed unless this is set.

Reply:
Hey Brian! The `showHeader` prop won't be necessary once [this PR ](https://github.com/marklogic/marklogic-data-hub/pull/6380#pullrequestreview-839597545)is merged, so I'll keep it as it is since it will show the header by default